instrumentation/undertow-1.4/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/undertow/UndertowSingletons.java

```diff
@@ -16,6 +18,9 @@ public final class UndertowSingletons {
 
   private static final Instrumenter<HttpServerExchange, HttpServerExchange> INSTRUMENTER;
 
+  public static final VirtualField<Runnable, PropagatedContext> RUNNABLE_PROPAGATED_CONTEXT =
```

Comment:
[minor] for virtual fields the naming pattern we followed so far is to name them with the type of their target, so in this case it would be `PROPAGATED_CONTEXT`.

Reply:
Got it. I've just renamed it.